examples/electron/package.json

```diff
@@ -40,7 +40,7 @@
     "prepare": "yarn run clean && yarn build",
     "clean": "theia clean",
     "build": "theia build",
```

Comment:
please add `--mode development` here too, in Theia repo we are never interested in building production-ready artifacts, the same for the browser example